addons/account_edi_ubl_cii/models/res_partner.py

```diff
@@ -40,81 +40,84 @@ class ResPartner(models.Model):
+            ('9926', "Bulgaria VAT - 9926"),
+            ('9934', "Croatia VAT - 9934"),
+            ('9928', "Cyprus VAT - 9928"),
+            ('9929', "Czech Republic VAT - 9929"),
+            ('0096', "Denmark CVR - 0096"),
+            ('0184', "Denmark CVR - 0184"),
+            ('0198', "Denmark SE - 0198"),
+            ('0191', "Estonia Company code - 0191"),
+            ('9931', "Estonia VAT - 9931"),
+            ('0037', "Finland LY-tunnus - 0037"),
+            ('0216', "Finland OVT code - 0216"),
+            ('0213', "Finland VAT - 0213"),
+            ('0002', "France SIRENE - 0002"),
+            ('0009', "France SIRET - 0009"),
+            ('9957', "France VAT - 9957"),
+            ('0204', "Germany Leitweg-ID - 0204"),
+            ('9930', "Germany VAT - 9930"),
+            ('9933', "Greece VAT - 9933"),
+            ('9910', "Hungary VAT - 9910"),
+            ('0196', "Iceland Kennitala - 0196"),
+            ('9935', "Ireland VAT - 9935"),
+            ('0211', "Italia Partita IVA - 0211"),
+            ('0097', "Italia FTI - 0097"),
+            ('0188', "Japan SST - 0188"),
+            ('0221', "Japan IIN - 0221"),
+            ('9939', "Latvia VAT - 9939"),
+            ('9936', "Liechtenstein VAT - 9936"),
+            ('0200', "Lithuania JAK - 0200"),
+            ('9937', "Lithuania VAT - 9937"),
+            ('9938', "Luxemburg VAT - 9938"),
+            ('9942', "Macedonia VAT - 9942"),
+            ('0230', "Malaysia - 0230"),
+            ('9943', "Malta VAT - 9943"),
+            ('9940', "Monaco VAT - 9940"),
+            ('9941', "Montenegro VAT - 9941"),
+            ('0106', "Netherlands KvK - 0106"),
+            ('0190', "Netherlands OIN - 0190"),
+            ('9944', "Netherlands VAT - 9944"),
+            ('0192', "Norway Org.nr. - 0192"),
+            ('9945', "Poland VAT - 9945"),
+            ('9946', "Portugal VAT - 9946"),
+            ('9947', "Romania VAT - 9947"),
+            ('9948', "Serbia VAT - 9948"),
+            ('0195', "Singapore UEN - 0195"),
+            ('9949', "Slovenia VAT - 9949"),
+            ('9950', "Slovakia VAT - 9950"),
+            ('9920', "Spain VAT - 9920"),
+            ('0007', "Sweden Org.nr. - 0007"),
+            ('9955', "Sweden VAT - 9955"),
+            ('9927', "Swiss VAT - 9927"),
+            ('0183', "Swiss UIDB - 0183"),
+            ('9952', "Turkey VAT - 9952"),
+            ('9932', "United Kingdom VAT - 9932"),
+            ('9959', "USA EIN - 9959"),
+            ('0060', "DUNS Number - 0060"),
+            ('0088', "EAN Location Code - 0088"),
+            ('0130', "Directorates of the European Commission - 0130"),
+            ('0135', "SIA Object Identifiers - 0135"),
+            ('0142', "SECETI Object Identifiers - 0142"),
+            ('0193', "UBL.BE party identifier - 0193"),
+            ('0199', "Legal Entity Identifier (LEI) - 0199"),
+            ('0201', "Codice Univoco UnitÃ  Organizzativa iPA - 0201"),
```

Comment:
There is a typo here `Unità`

Reply:
:o thanks, I'll fix that in another PR since it's already in 18.0